src/model/value-objects/invalidity.ts

```diff
@@ -0,0 +1,3 @@
+export interface Invalidity {
```

Comment:
Currently no, but the concept of invalidity is not unique to Segments so I could see it happen.
That being said, the main reason I created the separate interface/file was so that I could use the `Invalidity` as a return type.